test/moves/protect.test.ts

```diff
@@ -130,4 +131,36 @@ describe("Moves - Protect", () => {
     expect(enemyPokemon).toHaveMoveResult(MoveResult.SUCCESS);
     expect(leadPokemon).toHaveMoveResult(MoveResult.FAIL);
   });
+
+  it.each([
+    [1 / 3, 1],
+    [1 / 9, 2],
+    [1 / 27, 3],
+  ])(
+    "should have a success rate of %d after being used %d consecutive time(s) successfully",
```

Comment:
Here is a really useless suggestion.
If you wanna add the `s` for `times` conditionally :D

```suggestion
  it.each([
    [1 / 3, 1, ""],
    [1 / 9, 2, "s"],
    [1 / 27, 3, "s"],
  ])(
    "should have a success rate of %d after being used %d consecutive time%s successfully",
```

Feel free to do with that what you want

Reply:
I prefer the current version. Less clutter in the input arrays 😅